pkg/operator/status.go

```diff
@@ -35,24 +34,13 @@ func (c *serviceCAOperator) setFailingStatus(operatorConfig *operatorv1.ServiceC
 		})
 }
 
-func (c *serviceCAOperator) setProgressingStatus(operatorConfig *operatorv1.ServiceCA, message string) {
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions,
-		operatorv1.OperatorCondition{
-			Type:    operatorv1.OperatorStatusTypeProgressing,
-			Status:  operatorv1.ConditionTrue,
-			Message: message,
-		})
-
+func (c *serviceCAOperator) setProgressingStatus(operatorConfig *operatorv1.ServiceCA, reason, message string) {
 	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorv1.OperatorCondition{
-		Type:   operatorv1.OperatorStatusTypeFailing,
-		Status: operatorv1.ConditionFalse,
+		Type:    operatorv1.OperatorStatusTypeProgressing,
+		Status:  operatorv1.ConditionTrue,
+		Reason:  reason,
+		Message: message,
 	})
-
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions,
-		operatorv1.OperatorCondition{
-			Type:   operatorv1.OperatorStatusTypeAvailable,
-			Status: operatorv1.ConditionFalse,
-		})
 }
```

Comment:
@stlaz I think for setProgressing, we should only set Progressing==True, leave Available as/is.  wdyt? I wish there was a doc to explain what the correct behavior should be.  So, Progressing&Available may be true, however, only if Available=True &&Progressing=False do we report a new version.  That's what I'm thinking.